cmd/tsdbrelay/doc.go

```diff
@@ -22,15 +22,16 @@ Usage:
 
 The flags are:
 	-b="bosun"
-		Target Bosun server. Can specify port with host:port.
+		Target Bosun server. Can specify as host, host:port, or https://host:port.
 	-t=""
-		Target OpenTSDB server. Can specify port with host:port.
+		Target OpenTSDB server. Can specify as host, host:port or https://host:port.
```

Comment:
we do... if protocol is omitted it defaults to http. I yanked the parsing function we use in Scollector/bosun